### PR TITLE
Bootstrap 5.3 final

### PR DESCRIPTION
**Pull request by petschki**

### CODE REVIEW

**Comment by petschki** [comment]
@jenkins-plone-org please run jobs

File: news/333.feature [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Update to Bootstrap 5.2 final release.
```
Comment (mauritsvanrees) [inline news/333.feature:1]:
```suggestion
Update to Bootstrap 5.3 final release.
```

File: HOWTO_DEVELOP.rst [hunk 2]
```diff
@@ -1,21 +1,26 @@
-In order to develop plonetheme.barceloneta you need to execute on the root of
-the package::
+In order to develop plonetheme.barceloneta, make sure you use a ``node`` version >= 18
+and execute on the root of the package::
 
-     npm install
+     yarn install
```
Comment (mauritsvanrees) [inline HOWTO_DEVELOP.rst:4]: Do we have known minimum or maximum versions of `yarn`?

**Comment by petschki** [comment]
@jenkins-plone-org please run jobs

File: HOWTO_DEVELOP.rst (hunk 2 above)
Comment (petschki) [reply HOWTO_DEVELOP.rst]: After discussion with johannes I've reverted back to "pure" npm.

**Pull request merged**